frontend_tests/node_tests/templates.js

```diff
@@ -1716,3 +1716,59 @@ run_test('recipient_row', () => {
     assert(html.indexOf('<script>alert("Hello")</script>') === -1);
     assert(html.indexOf('&lt;script&gt;alert(&quot;Hello&quot;)&lt;/script&gt;') !== -1);
 });
+
+run_test('invitation_failed_error', () => {
+    let err_list = [];
+    err_list.push("hamlet@zulip.com: Account has been deactivated.");
+    let data = {
+        error_message: "We weren't able to invite anyone.",
+        error_list: err_list,
+        is_invitee_deactivated: true,
+        is_admin: true,
+    };
+
+    let html = render('invitation_failed_error', data);
```

Comment:
Try this
```
    html = '<div>';
    html += render('invitation_failed_error', data);
    html += '<div>';
```

Reply:
@hackerkid thanks so much! it worked :tada: